spec/helpers/navbar_helper_spec.rb

```diff
@@ -83,29 +84,25 @@
           end
         end
 
-        it 'renders navbar with left and right items' do
-          fragment = view.tramway_navbar do |nav|
-            nav.left do
-              items.each do |(name, path)|
-                nav.item name, path
-              end
-            end
+        include_examples 'Helpers Navbar'
+      end
 
-            nav.right do
-              items.each do |(name, path)|
-                nav.item name, path
-              end
-            end
+      context 'with Tramway entities preset' do
+        before do
+          Tramway.configure do |config|
+            config.entities = [:user]
           end
+        end
 
-          expect(fragment).to have_css left_items_css
-          expect(fragment).to have_css right_items_css
+        it 'renders navbar with users link on the left' do
+          fragment = view.tramway_navbar
 
-          items.each do |(name, path)|
-            expect(fragment).to have_css "#{left_items_css} a[href='#{path}']", text: name
-            expect(fragment).to have_css "#{right_items_css} a[href='#{path}']", text: name
-          end
+          path = Rails.application.routes.url_helpers.users_path
```

Comment:
we can use `let` and move `fragment` and `path` from `it`

```rb
let(:fragment) { view.tramway_navbar }
let(:path) { Rails.application.routes.url_helpers.users_path }
```

Reply:
Fixed! Thanks!